src/Http/Routing/src/ShortCircuit/ShortCircuitMetadata.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.AspNetCore.Routing.ShortCircuit;
 
-internal sealed class ShortCircuitMetadata
+internal sealed class ShortCircuitMetadata : IShortCircuitMetadata
```

Comment:
There' no need for `IShortCircuitMetadata` interface. Since `ShortCircuitMetadata` is internal, I suggest to remove both `IShortCircuitMetadata` and `ShortCircuitMetadata` and only have `ShortCircuitAttribute` and update the middleware to work with the new attribute.